XamarinCommunityToolkit/Views/ISideMenuList.shared.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Collections.Generic;
+using Xamarin.Forms;
+
+namespace Microsoft.Toolkit.Xamarin.Forms.UI.Views
+{
+	public interface ISideMenuList<T> : IList<T> where T : View
+	{
+		void Add(View view, SideMenuPosition position);
```

Comment:
This adds an... item? To the menu? I mean, it's a view so it can be anything, right?

Reply:
Yes, it can be anything
sideMenuView.add(someView, SideMenuPosition.LeftMenu);
sideMenuView.add(someView, SideMenuPosition.RightMenu);